### PR TITLE
verify prow cron timezone

### PR DESCRIPTION
**Pull request by krzyzacy**
before we move rest of the large cluster jobs, just make sure

/assign @shyamjvs

### CODE REVIEW

**Comment by krzyzacy** [comment]
/assign @BenTheElder 
wanna give a quick stamp? (this is the borked kubemark job which I don't really care till bazel image is sorted out)

**Comment by BenTheElder** [comment]
Discussed offline, we can just enforce UTC in the cron code instead which is probably a good idea.

**Comment by krzyzacy** [comment]
/hold
adding some other code to cron client

**Comment by BenTheElder** [comment]
Per offline discussion, we think we want to only support UTC for now (which doesn't require tzdata etc...) which horologium should have already been using by default, but this should ensure that it does. We can remove the prefix later if we want to support timezeones and all the fun that comes with them.
/shrug

**Comment by krzyzacy** [comment]
let's see, I think we just made it more UTC
/hold cancel

File: prow/cron/cron.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ func (c *Cron) addPeriodic(p config.Periodic) error {
 
 // addJob adds a cron entry for a job to cronAgent
 func (c *Cron) addJob(name, cron string) error {
-	id, err := c.cronAgent.AddFunc(cron, func() {
+	id, err := c.cronAgent.AddFunc("TZ=UTC "+cron, func() {
```
Comment (shyamjvs) [inline prow/cron/cron.go]: Can you try this with our scalability release jobs you recently moved to prow? To verify if it's WAI.

Comment (krzyzacy) [reply prow/cron/cron.go]: https://k8s-testgrid.appspot.com/sig-testing-canaries#kubemark-canary-prow the canary job is WAI

Comment (shyamjvs) [reply prow/cron/cron.go]: Awesome!

Comment (krzyzacy) [reply prow/cron/cron.go]: sent https://github.com/kubernetes/test-infra/pull/5600 for rest of the scalability jobs :-)

**Pull request merged**